### ISSUE TITLE
ADDED new labels `qa-accessibility` project 

### ISSUE THREAD

**Issue by sushilshinde** [topcoder-platform/topcoder-react-ui-kit#12]
_This is not an issue_

Added new labels for `qa-accessibility` project

- Accessibility  
- Code Review Failed  
- Code Review Passed  
- QA Pass  
- Ready for QA
